src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IUsingStatement.cs

```diff
@@ -916,5 +916,1461 @@ public static void M1()
+Block[B2] - Block
+    Predecessors: [B1]
+    Statements (1)
+        IFlowCaptureOperation: 1 (OperationKind.FlowCapture, Type: null, IsInvalid, IsImplicit) (Syntax: 'GetDisposable()')
+          Value: 
+            IFlowCaptureReferenceOperation: 0 (OperationKind.FlowCaptureReference, Type: MyDisposable, IsInvalid, IsImplicit) (Syntax: 'GetDisposable()')
+
+    Next (Regular) Block[B4]
+Block[B3] - Block
+    Predecessors: [B1]
+    Statements (1)
+        IFlowCaptureOperation: 1 (OperationKind.FlowCapture, Type: null, IsInvalid, IsImplicit) (Syntax: 'input')
+          Value: 
+            IParameterReferenceOperation: input (OperationKind.ParameterReference, Type: MyDisposable, IsInvalid) (Syntax: 'input')
+
+    Next (Regular) Block[B4]
+Block[B4] - Block
+    Predecessors: [B2] [B3]
+    Statements (1)
+        IFlowCaptureOperation: 2 (OperationKind.FlowCapture, Type: null, IsInvalid, IsImplicit) (Syntax: 'GetDisposable() ?? input')
+          Value: 
+            IFlowCaptureReferenceOperation: 1 (OperationKind.FlowCaptureReference, Type: MyDisposable, IsInvalid, IsImplicit) (Syntax: 'GetDisposable() ?? input')
+
+    Next (Regular) Block[B5]
+        Entering: {R1} {R2}
+
+.try {R1, R2}
+{
+    Block[B5] - Block
+        Predecessors: [B4]
+        Statements (1)
+            IExpressionStatementOperation (OperationKind.ExpressionStatement, Type: null) (Syntax: 'b = true;')
+              Expression: 
+                ISimpleAssignmentOperation (OperationKind.SimpleAssignment, Type: System.Boolean) (Syntax: 'b = true')
+                  Left: 
+                    IParameterReferenceOperation: b (OperationKind.ParameterReference, Type: System.Boolean) (Syntax: 'b')
+                  Right: 
+                    ILiteralOperation (OperationKind.Literal, Type: System.Boolean, Constant: True) (Syntax: 'true')
+
+        Next (Regular) Block[B9]
+            Finalizing: {R3}
+            Leaving: {R2} {R1}
+}
+.finally {R3}
+{
+    Block[B6] - Block
+        Predecessors (0)
+        Statements (0)
+        Jump if True (Regular) to Block[B8]
+            IIsNullOperation (OperationKind.IsNull, Type: System.Boolean, IsInvalid, IsImplicit) (Syntax: 'GetDisposable() ?? input')
+              Operand: 
+                IFlowCaptureReferenceOperation: 2 (OperationKind.FlowCaptureReference, Type: MyDisposable, IsInvalid, IsImplicit) (Syntax: 'GetDisposable() ?? input')
+
+        Next (Regular) Block[B7]
+    Block[B7] - Block
+        Predecessors: [B6]
+        Statements (1)
+            IInvocationOperation (virtual void System.IDisposable.Dispose()) (OperationKind.Invocation, Type: System.Void, IsInvalid, IsImplicit) (Syntax: 'GetDisposable() ?? input')
+              Instance Receiver: 
+                IConversionOperation (TryCast: False, Unchecked) (OperationKind.Conversion, Type: System.IDisposable, IsInvalid, IsImplicit) (Syntax: 'GetDisposable() ?? input')
+                  Conversion: CommonConversion (Exists: True)
```

Comment:
>(Exists: True) [](start = 47, length = 14)

This is weird. I'm not sure how we'd go about rectifying it without having language-specific information, but it's very weird that this conversion should "Exist" when it very clearly does not, and the fact that it does not is the error in this code.

Reply:
> This is weird. I'm not sure how we'd go about rectifying it without having language-specific information, but it's very weird that this conversion should "Exist" when it very clearly does not, and the fact that it does not is the error in this code.

I really don't think this makes any difference for the analysis that will be done on the graph. Not a single bit.

---
In reply to: [178671855](https://github.com/dotnet/roslyn/pull/25828#discussion_r178671855) [](ancestors = 178671855)

Reply:
FWIW, there is an IsInvalid indicator on the node.

---
In reply to: [178673415](https://github.com/dotnet/roslyn/pull/25828#discussion_r178673415) [](ancestors = 178673415,178671855)